### DISCUSSION TITLE
Exclude traffic from IP address

### DISCUSSION THREAD

**Discussion by filippodanesi** [plausible/analytics#2015] [Anything Else]
Hi everyone! 👋
I state that I am new to using Plausible.
I was wondering if there was the possibility to exclude IP addresses from the tracking (as with the Google Analytics filters), in order to exclude traffic from my pc or the work one.

Thank you 😁

**Comment by RobertJoonas** [comment]
Hi @filippodanesi!

Excluding visits by IP address is not possible unfortunately, but we do have other ways to exclude yourself from being tracked:
1) localStorage flag: https://plausible.io/docs/excluding-localstorage
2) using an adblocker: https://plausible.io/docs/excluding

Hope this helps!

**Comment by metmarkosaric** [comment]
**Update**: It’s now easy to exclude your internal traffic from being counted in your stats. See how here: https://plausible.io/docs/excluding

**Reply by atilladeniz** [reply]
Is this for Self hosting also possible because I don't see the options in the settings..

**Reply by ruslandoga** [reply]
👋 @atilladeniz 

This feature is available at `/<site name>/settings/shields/ip_addresses` in CE. What version of Plausible are you running?

**Reply by atilladeniz** [reply]
Oh, I understand now! I see the new CE has these settings. Thank you @ruslandoga for taking the time to help me and for your detailed answer!

**Comment by satonotdead** [comment]
Hi there, I can't found this option on my instance.

What's going wrong? I'm self-hosting :)

**Reply by ruslandoga** [reply]
👋 @satonotdead 

Please make sure you are using the latest version. Current latest version is v2.1.4. See https://github.com/plausible/community-edition for more details :)

**Comment by satonotdead** [comment]
Thanks for your reply, I see that I need to tweak my `docker-compose.yml` file a little in order to get the last updated version :)
